webpack/scenes/ContentViews/Details/ContentViewDetailActions.js

```diff
@@ -85,6 +85,16 @@ export const createContentViewFilter = (cvId, params) => post({
   errorToast: error => __(`Something went wrong while creating the filter! ${getResponseErrorMsgs(error.response)}`),
 });
 
+export const createPackageFilterRule = (filterId, params) => post({
```

Comment:
We need to figure out which of these we want to keep: 

https://github.com/Katello/katello/pull/9536/files#diff-21a7ef68cbc8178319384d3863701da3b9758de59b6b8d6f3d1bbbe837c143f9R157-R165

Because the route is the same for multiple rule types.

Reply:
Rebased on your changes..